filebeat/prospector/udp/harvester.go

```diff
@@ -0,0 +1,52 @@
+package udp
+
+import (
+	"net"
+	"time"
+
+	"github.com/elastic/beats/filebeat/channel"
+	"github.com/elastic/beats/filebeat/util"
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/logp"
+)
+
+type Harvester struct {
+	outlet channel.Outleter
+	done   chan struct{}
+}
+
+func (h *Harvester) Run() {
+	listener, err := net.ListenPacket("udp", ":8080")
+
+	if err != nil {
+		logp.Err("error happened: %s", err)
+		return
+	}
+	defer listener.Close()
+
+	//logp.Info("Now listening for logs via %s on %s", ll.config.Protocol, address)
+	buffer := make([]byte, 1000)
```

Comment:
@ruflin I realize in theory you could have up to 65507 bytes of data in IPv4 (removing UDP and IP header overhead of 20 and 8 bytes respectively) although I know that you risk having fragmentation of this because of path MTU (which I guess is generally between 1400 and 1500 - not taking into account jumbo frames).  Considering this should only be done locally on the server, i figured this shouldn't happen, so thus why I chose a slightly larger 2048 or 4096 bytes buffer size.  

I know many applications  tend to limit to <= 1024 bytes for a UDP buffer (especially for cross-host communication), although from what I've tested so far, I haven't had messages that were cut off, but I'm aware this could happen, especially if you're going form one host to another even on the same rack.  You could potentially stick with the 1024 byte buffer as a default but allow users to specify the port and the buffer size via the prospector config parameters.

I think it would maybe be nice to have someone else give some feedback on this, especially from the networking stack side to confirm if this is a reasonable assumption or not.  What do you think?